lib/js/stanford_news_list.behavior.js

```diff
@@ -13,9 +13,8 @@ export default {
   attach(context, settings) {
     (function ($) {
       $(".collapsable-menu", context).click(function() {
-        $(this, context).toggleClass("show");
+        $(this).toggleClass("show");
```

Comment:
This is a special case when using the `this` keyword already in a context selector.